Client/mods/deathmatch/logic/CClientObjectManager.h

```diff
@@ -25,6 +25,9 @@ class CClientObjectManager
 
     void DeleteAll();
 
+    std::vector<CClientObject*>::const_iterator IterBegin() { return m_Objects.begin(); };
+    std::vector<CClientObject*>::const_iterator IterEnd() { return m_Objects.end(); };
```

Comment:
Ah, so you just added these methods? Then just change this to my suggestion I guess:
```cpp
const auto& GetObjects() { return m_Objects; }
```

Reply:
I think they want to keep `m_Objects` private, assuming it's already currently private.

Reply:
With a `const auto&` it'll remain basically private. (you can do the same with 2 iterators as with a `const std::vector&`. But a range based loop is way more readable than an iter one)

Reply:
I think the method should be also marked **const**:

```const auto& GetObjects() const { return m_Objects; }```